ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/MoneyMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.utils.MonetaryConversions;
+
+import java.math.BigDecimal;
+
+import static java.util.Objects.nonNull;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+
+@Component
+public class MoneyMapper implements Mapper<String, BigDecimal> {
+
+    @Override
+    public String to(BigDecimal amountInPounds) {
```

Comment:
minor suggestion im thinking if we have generic name `MoneyMapper`, maybe we should have more explicit name for methods like `toPenniesFromPounds`, `fromPenniesToPounds` or call class `PenniesAndPoundsMapper` mmmm typed it then don't like it... 😄

Reply:
I came to comment the same what @markry-kainos did, better to change it to toPennies and fromPounds or like what Mark suggested.

Reply:
I was about to comment on this. Glad Mark has done it already. Please change them to proper method names

Reply:
I made it consistent with others to implement the contract we have defined in interface `Mapper `